com.vcvrack.Rack2.yaml

```diff
@@ -0,0 +1,76 @@
+id: com.vcvrack.Rack2
+runtime: org.freedesktop.Platform
+runtime-version: &runtime-version '23.08'
+sdk: org.freedesktop.Sdk
+command: rack
+
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=all
+  - --allow=bluetooth
```

Comment:
why is `all` devices and bluetooth needed ?

Reply:
Fixed